clab/config/template.go

```diff
@@ -33,7 +33,11 @@ type NodeConfig struct {
 // Load templates from all paths for the specific role/kind
 func LoadTemplates(tmpl *template.Template, role string) error {
 	for i := range TemplatePaths {
-		fn := filepath.Join(TemplatePaths[i], fmt.Sprintf("*__%s.tmpl", role))
+		p := TemplatePaths[i]
```

Comment:
@kellerza any specific reason for not using

```go
for _, p := range TemplatePaths {
}
```
(eliminating the extra step of `p := TemplatePaths[i]`)

Reply:
I changed that in 16c6b58 to speed up the merging